### ISSUE TITLE
Get Recommended Books in the Dashboard

### ISSUE THREAD

**Issue by Mohitmadhav** [Project-Easter/Flutter-UI#136]
## Description
- Users should get recommended books according to genres selected by them in the user preferences dialog. 
- The User Preferences dialog is being resolved here: #130. So for now, take any 3 random genres.
- Another section like *Discover New* (As shown in the screenshot) should be created containing books based on these 3 random genres because the user preferences tab is being fixed.

## Screenshots
<div class ="row"> 
<img src ="https://user-images.githubusercontent.com/59333817/136596187-58f9b9ac-577b-4cff-bf28-6126d270dd27.png" width = 48%/>
<img src ="https://user-images.githubusercontent.com/59333817/136596389-d9bf9ebc-38c0-4e14-8be5-d0c577c3b73e.png" width = 48%/>
</div>

## Steps to Reproduce
Please provide detailed steps for reproducing the issue.

1. Genre files: F:\Career\Flutter\Explr\Flutter-UI\lib\constants\genres.dart
2. Books should be fetched from the Google Books API  according to any 3 random genres.
3. Database files: lib\services\database_service.dart

## Additional data
Add your working screenshots in the PR. And make sure to follow the README guidelines.

**Comment by keyurgit45** [comment]
can I work on this? @Mohitmadhav

**Comment by Mohitmadhav** [comment]
Will assign it to you once I merge #146

**Comment by Mohitmadhav** [comment]
Either we can try out this: https://pub.dev/packages/googleapis

Or we'll have to change the Books API but I think every other free API will give similar issues.

**Comment by Mohitmadhav** [comment]
You can look into it @keyurgit45

**Comment by keyurgit45** [comment]
@Mohitmadhav We can use the search parameter in query to get books from the given category. Also, Do we have show books from a favorite author?

**Comment by Mohitmadhav** [comment]
No no, just the books based on the genres selected by the user for now.

**Comment by keyurgit45** [comment]
@Mohitmadhav Google books API is not working as we want. If I want books related to the fiction genre, the response is giving me other genres also. I don't know why it is happening. Also if the user selected 5 genres then we have to make 5 different requests to the API so it would be a quite inefficient task.

**Comment by Mohitmadhav** [comment]
No problem @keyurgit45, lets's focus on other issues right now. I am planning to release the app at the earliest.

**Comment by keyurgit45** [comment]
Hey @Mohitmadhav I think we need to look into this first. Can you create a new issue for this?
![image](https://user-images.githubusercontent.com/75253653/171593268-a0903010-3781-4f53-8eb6-83ba767b6d6c.png)

**Comment by Mohitmadhav** [comment]
> @Mohitmadhav I think we need to look into this first. Can you create a new issue for this? ![image](https://user-images.githubusercontent.com/75253653/171593268-a0903010-3781-4f53-8eb6-83ba767b6d6c.png)

Yeah you can create one @keyurgit45 but this isn't being displayed in my console so do have a look again
